modules/documents/app/forms/document_form.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+class DocumentForm < ApplicationForm
+  form do |f|
+    f.select_list(
+      name: :category_id,
+      label: I18n.t("label_document_category"),
+      input_width: :medium,
+      required: true
+    ) do |select|
+      DocumentCategory.find_each do |category|
+        select.option(value: category.id, label: category.name)
+      end
+    end
+
+    f.text_field(
+      name: :title,
+      label: I18n.t("label_document_title"),
+      required: true
+    )
+
+    if OpenProject::FeatureDecisions.block_note_editor_active? && model.category&.name == "Experimental"
```

Comment:
Can we enable this for all categories?

Reply:
This is intentional.

The idea is that we don't affect projects where the Documents module is active. Mostly because BlockNote & CKEditor might have some differences and this could cause some issues with the data persisted to the database.